site/src/content/docs/ref/init-package.mdx

```diff
@@ -134,58 +145,166 @@ package:
+
+:::caution
+
+(Linux only) Deploying the 'k3s' component will require `root` access (not just `sudo`), as it modifies your host machine to install the cluster and by default only provides access to the cluster to the `root` user.
+
+:::
 
 :::tip
 
-Given that these flows are so similar you actually can `zarf package deploy` the init package.  This is similar to accepting the defaults for any init-specific command flags and is useful when trying to deploy packages in a more generic way.
+The `k3s` component included differs from the default `k3s` install in that it disables the installation of `traefik` out of the box.  This was done so that people could more intentionally choose if they wanted `traefik` or another ingress provider (or no ingress at all) depending on their needs.  If you would like to return `k3s` to its defaults, you can set the `K3S_ARGS` Zarf variable to an empty string:
+
+```text
+root@machine ~ # zarf init --components k3s --set K3S_ARGS="" --confirm
+```
+
+:::
+
+:::tip
+
+You can further customize how the git-server behaves by setting variables such as `GIT_SERVER_PVC_SIZE` with a [config file](/ref/config-files/) or `--set` on `zarf init`.
+
+To see a full list of `variables` you can view the [zarf.yaml that defines the git-server](https://github.com/defenseunicorns/zarf/blob/main/packages/gitea/zarf.yaml).
+
+:::
+
+## Putting it All Together
+
+The package definition 'init' is similar to writing any other Zarf Package, but with a few key differences:
+
+Starting with `kind` and `metadata`:
+
+```yaml {3, 6, 9}
+# zarf.yaml
+# kind must be ZarfInitConfig
+kind: ZarfInitConfig
+metadata:
+  # name *can* be anything, but it is generally recommended to end with 'init'
+  name: init
+  # version should be empty as it will be set by the Zarf CLI
+  # (this is ONLY for the 'init' package)
+  # version: 0.1.0
+...
+```
+
+In order for Zarf to operate correctly, the following `components`:
+
+- must be defined, ordered, and named **exactly** as shown below
+- must have the `required` field set to `true`
+
+```yaml {3, 7, 11, 15}
+# zarf.yaml
+components:
+  # components (like k3s) that spin up a cluster...
+
+  - name: zarf-injector
+    required: true
+    ...
+
+  - name: zarf-seed-registry
+    required: true
+    ...
+
+  - name: zarf-registry
+    required: true
+    ...
+
+  - name: zarf-agent
+    required: true
+    ...
+
+  # optional components that need a cluster ...
+```
+
+:::note
+
+In order to reproduce the following example, you will need to have the Zarf repository cloned locally.
+
+```bash
+git clone https://github.com/defenseunicorns/zarf.git
+cd zarf
+mv zarf.yaml zarf.yaml.bak
+cat <<EOF > zarf.yaml
+...
+EOF
+```
+
+You can learn more about creating a custom init package in the [Creating a Custom 'init' Package Tutorial](/tutorials/8-custom-init-packages).
```

Comment:
IMO this note is a bit confusing. I think the average user will be unsure if this section is about how Zarf interacts with init packages or if it's about how to create your own init packages. I also don't think they'll know what to put in the ellipsis.